discovery-provider/alembic/trigger_sql/handle_repost.sql

```diff
@@ -69,7 +69,7 @@ begin
         'milestone:' || milestone_name  || ':id:' || new.repost_item_id || ':threshold:' || milestone,
         new.blocknumber,
         new.created_at,
-        json_build_object('type', milestone_name, 'threshold', milestone)
+        json_build_object('type', milestone_name, new.repost_type::text || '_id', new.repost_item_id, 'threshold', milestone)
```

Comment:
confirming `new.repost_type` and `new.save_type` will always be lower case? could we lower case `new.repost_type::text` in case? also if a playlist is an album the repost_type is still `playlist` right?

Reply:
i guess another alternative would be adding `entity_id` and `entity_type` to the json instead if you're trying to make it work for any music type